pkg/cmd/buildinformation/bulkdelete/bulk-delete.go

```diff
@@ -0,0 +1,92 @@
+package bulkdelete
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/MakeNowJust/heredoc/v2"
+	"github.com/OctopusDeploy/cli/pkg/cmd"
+	"github.com/OctopusDeploy/cli/pkg/constants"
+	"github.com/OctopusDeploy/cli/pkg/factory"
+	"github.com/OctopusDeploy/cli/pkg/output"
+	"github.com/OctopusDeploy/cli/pkg/question"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/buildinformation"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/client"
+	"github.com/spf13/cobra"
+)
+
+type DeleteOptions struct {
+	*cmd.Dependencies
+	*question.ConfirmFlags
+	IDs []string
+}
+
+func NewDeleteOptions(ids []string, dependencies *cmd.Dependencies, confirmFlags *question.ConfirmFlags) *DeleteOptions {
+	return &DeleteOptions{
+		Dependencies: dependencies,
+		ConfirmFlags: confirmFlags,
+		IDs:          ids,
+	}
+}
+
+func NewCmdBulkDelete(f factory.Factory) *cobra.Command {
+	confirmFlags := question.NewConfirmFlags()
+	cmd := &cobra.Command{
+		Use:   "bulk-delete",
+		Short: "Bulk delete build information",
+		Long:  "Bulk delete build information in Octopus Deploy",
+		Example: heredoc.Docf(`
+			$ %[1]s build-information bulk-delete BuildInformation-1 BuildInformation-2
```

Comment:
This feels a bit weird. 
A bulk operation would make sense if I could specify every version for a given package id or a set of versions